doc/tutorial/tutorial.tex

```diff
@@ -3581,6 +3582,91 @@ \subsection*{Notes}
+                \item{ {\tt W.win}  {\it The {\tt wannier90} input file}}
+        \end{itemize}
+\end{itemize}
+
+\begin{enumerate}
+	\item Run \pwscf\ to obtain the ground state of tungsten\\
+	{\tt pw.x -in W.scf > scf.out}
+	
+	\item Run \pwscf\ to obtain the Bloch states on a $10\times10\times10$ uniform $k$-point
+	grid\\ 
+	{\tt pw.x -in W.nscf > nscf.out}
+	
+	\item Run \wannier\ to generate a list of the required overlaps (written
+	into the {\tt W.nnkp} file)\\
+	{\tt wannier90.x -pp W}
+	
+        \item Run {\tt projwfc} to compute the projectabilities of the Bloch states
+              onto the Bloch sums obtained from the PAOs in the pseudopotential \\
+        {\tt projwfc.x -in W.proj > proj.out}
+
+        \item Run {\tt generate\_weights} to extract the projectabilitites from {\tt proj.out}
+              in a format suitable to be read by Xmgrace or gnuplot \\
+	{\tt ./generate\_weights.sh}
+
+        \item Plot the projectabilities and fit the data with the complementary error function
+              $$f(\epsilon;\mu,\sigma) = \frac{1}{2}\mathrm{erfc}(-\frac{\mu - \epsilon}{\sigma}).$$
+		We are going Xmgrace to plot the projectabilities and perform the fitting. Open Xmgrace \\
+	{\tt xmgrace }
+		
+	To Import the {\tt p\_vs\_e.dat} file, click on {\tt Data} from the top bar and then {\tt Import -> ASCII...}.
+	At this point a new window {\tt Grace: Read sets} should pop up . Select {\tt p\_vs\_e.dat} in the {\tt Files} section, click {\tt Ok} at the bottom and close the window. You should now be able to see a quite noisy function that is bounded between 1 and 0. You can modify the appearence of the plot by clicking on {\tt Plot} in the top bar and then {\tt Set appearance...}. In the {\tt Main} section of the pop-up window change the symbol type from {\tt None} to {\tt Circle} and symbol size from 100 to 25. Also change symbol color from black to blue. Change the line type from straight to none. Move to the {\tt Symbols} section and change the filling pattern from none to full (indentified by a full black square) and then close the window. For the fitting, go to {\tt Data -> Transformations -> Non-linear curve fitting}. In this window, select the source from the {\tt Set} box and in the {\tt Formula} box insert the following \\
+	
+	{\tt y = 0.5 * erfc( ( x - A0 ) / A1 )}  \\
+
+Select 2 as number of parameters, give 40 as initial condition for {\tt A0} and 7 for {\tt A1}. Click {\tt Apply}. A new window should pop up with the stats of the fitting. In particular you should find a {\tt Correlation coefficient} of 0.96 and a value of $39.9756$ for {\tt A0} and $6.6529$ for {\tt A1}. These are the value of $\mu_{fit}$ and $\sigma_{fit}$ we are going to use for the SCDM method. In particular, $\mu_{SCDM} = \mu_{fit} - 3\sigma_{fit} = 20.0169$ eV and $\sigma_{SCDM} = \sigma_{fit} = 6.6529$ eV. You should now see the fitting function, as well as the projectabilities, in the graph (see Fig. \ref{fig:W_fit}-(a)).\\
```

Comment:
Explain that the specific choice `\mu_{SCDM} = \mu_{fit} - 3\sigma_{fit}` and `\sigma_{SCDM} = \sigma_{fit}` has been introduced in Ref. [3] and that there is the motivation and validation in there. Maybe somewhere at the end of the exercise say to cite [3] if you use this approach to get `mu` and `sigma`?